examples/svm/plot_svm_regression.py

```diff
@@ -23,22 +23,51 @@
 
 # #############################################################################
 # Fit regression model
-svr_rbf = SVR(kernel='rbf', C=1e3, gamma=0.1)
-svr_lin = SVR(kernel='linear', C=1e3)
-svr_poly = SVR(kernel='poly', C=1e3, degree=2)
+svr_rbf = SVR(kernel='rbf', C=100, gamma=0.1, epsilon=.1)
+svr_lin = SVR(kernel='linear', C=100)
+svr_poly = SVR(kernel='poly', C=100, degree=3, epsilon=.1, coef0=1)
 y_rbf = svr_rbf.fit(X, y).predict(X)
 y_lin = svr_lin.fit(X, y).predict(X)
 y_poly = svr_poly.fit(X, y).predict(X)
 
 # #############################################################################
 # Look at the results
 lw = 2
-plt.scatter(X, y, color='darkorange', label='data')
-plt.plot(X, y_rbf, color='navy', lw=lw, label='RBF model')
-plt.plot(X, y_lin, color='c', lw=lw, label='Linear model')
-plt.plot(X, y_poly, color='cornflowerblue', lw=lw, label='Polynomial model')
-plt.xlabel('data')
-plt.ylabel('target')
-plt.title('Support Vector Regression')
-plt.legend()
+fig, (ax1, ax2, ax3) = plt.subplots(1, 3, figsize=(15, 10), sharey=True)
+
+ax1.plot(X, y_rbf, color='m', lw=lw, label='RBF model')
```

Comment:
Could we please make use of a loop to avoid this repeated code now that we have three separate plots?

Reply:
Hi @jnothman. I created a loop for the plots, committed the code, and passed testing. Please let me know if there is anything else.